examples/webgpu_sandbox.html

```diff
@@ -55,6 +55,15 @@
 				scene = new THREE.Scene();
 				scene.background = new THREE.Color( 0x222222 );
 
+				//
+
+				renderer = new WebGPURenderer( { requiredFeatures: [ 'texture-compression-bc', 'texture-compression-astc' ] } );
```

Comment:
This is also something that might cause troubles. You have specify the required features (e.g. texture compression formats) _before_ the adapter of `WebGPURenderer`has been created. 

A viewer application would have to know it advance what type of assets are going to be imported so it can request the respective features (which is of course unrealistic). I wonder now if it's best to always requests all supported features (that includes texture compression formats) every time `WebGPURenderer` is created. Something like the following in `WebGPURenderer.init()`:

```js
const gpuAdapter = await navigator.gpu.requestAdapter(); // dummy adapter for feature detection

const features = [];

for ( const name of featureNames ) {

    if ( gpuAdapter.features.has( name ) ) {

        features.push( name );

    }

}

const deviceDescriptor = {
    requiredFeatures: features
};

const device = await adapter.requestDevice( deviceDescriptor ); // real adapter of WebGPURenderer

```